parsigs/parse_sig_api.py

```diff
@@ -338,16 +355,33 @@ def _get_amount_from_frequency_tags(frequency):
             return 2
 
 
+
 def _get_times_from_latin(frequency):
     latin_freq = _get_latin_frequency(frequency)
     if latin_freq:
         return latin_freq.times
 
 
+def _get_interval_from_latin(frequency):  # for cases like qxd/qxh (the interval changes and times is 1)
+    latin_freq = _get_latin_frequency(frequency)
+    if latin_freq:
+        return latin_freq.interval
+    else:
+        return 1
+
+
 def _get_latin_frequency(frequency):
     for latin_freq in latin_frequency_types.keys():
-        if latin_freq in frequency:
-            return latin_frequency_types[latin_freq]
+        final_freq = latin_freq.lower().replace('.', '')
+        if final_freq in frequency:
```

Comment:
this check is redundant in the best case and buggy in the worst if frequency has `.` in it